lib/vtls/schannel.c

```diff
@@ -2754,6 +2754,148 @@ static void *schannel_get_internals(struct ssl_connect_data *connssl,
   return &backend->ctxt->ctxt_handle;
 }
 
+HCERTSTORE schannel_get_cached_cert_store(struct Curl_cfilter *cf,
+                                          const struct Curl_easy *data)
+{
+  struct Curl_multi *multi = data->multi_easy ? data->multi_easy : data->multi;
+
+  if(!multi) {
+    return NULL;
+  }
+
+  if(!multi->ssl_backend_data) {
+    return NULL;
+  }
+
+  struct schannel_multi_ssl_backend_data *mbackend;
```

Comment:
We write C89 code, so we prefer the variable declarations at the top of every code block. Even for this code, I presume there is no compiler that is going to warn about it in this windows specific section.

Reply:
OK